integration-tests/gatsby-cli/gatsby-sites/gatsby-build-errors/package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "gatsby-starter-default",
+  "private": true,
+  "description": "A simple starter to get up and developing quickly with Gatsby",
+  "version": "0.1.0",
+  "author": "Kyle Mathews <mathews.kyle@gmail.com>",
+  "dependencies": {
+    "gatsby": "2.23.11-dev-1593442963379",
```

Ops, sorry I missed this in initial review

```suggestion
    "gatsby": "^2.23.11",
```